cms/templates/cms/home_page.html

```diff
@@ -75,7 +75,46 @@
         </div>
       </div>
 
-            <section class="row current-programs-section">
+      <div class="row how-it-works-section">
```

Comment:
Is there a particular reason why you're using `<div>` rather than `<section>`? The latter is more semantically meaningful.